test-programs/account-compression-test/tests/address_merkle_tree_tests.rs

```diff
@@ -147,6 +147,67 @@ async fn test_address_queue_and_tree_functional_custom() {
     }
 }
 
+async fn test_address_queue_and_tree_invalid_size<R: RpcConnection>(
+    context: &mut R,
+    payer: &Keypair,
+    merkle_tree_keypair: &Keypair,
+    queue_keypair: &Keypair,
+    merkle_tree_config: &AddressMerkleTreeConfig,
+    queue_config: &AddressQueueConfig,
+    merkle_tree_size: usize,
+    queue_size: usize,
+    expected_error_code: u32,
+) {
+    let queue_account_create_ix = create_account_instruction(
+        &payer.pubkey(),
+        queue_size,
+        context
+            .get_minimum_balance_for_rent_exemption(queue_size)
+            .await
+            .unwrap(),
+        &account_compression::ID,
+        Some(&queue_keypair),
+    );
+    let mt_account_create_ix = create_account_instruction(
+        &payer.pubkey(),
+        merkle_tree_size,
+        context
+            .get_minimum_balance_for_rent_exemption(merkle_tree_size)
+            .await
+            .unwrap(),
+        &account_compression::ID,
+        Some(&merkle_tree_keypair),
+    );
+
+    let instruction =
+        account_compression::sdk::create_initialize_address_merkle_tree_and_queue_instruction(
+            0,
+            payer.pubkey(),
+            payer.pubkey(),
+            None,
+            merkle_tree_keypair.pubkey(),
+            queue_keypair.pubkey(),
+            merkle_tree_config.clone(),
+            queue_config.clone(),
+        );
+    let c_ix =
+        solana_sdk::compute_budget::ComputeBudgetInstruction::set_compute_unit_limit(10_000_000);
```

Comment:
```suggestion
```

Reply:
done

Reply:
there is no need to have the instruction at all the env is already inited with the compute unit limit at 1.4m
We can keep it since it doesn't do anything bad but in general.

Reply:
removed